src/balloons/ghost.ts

```diff
@@ -0,0 +1,36 @@
+import Default, { BalloonOptions } from './default';
+import { random } from '@/utils';
+
+type GhostOptions = BalloonOptions & {
+  fadeSpeed: number | [number, number];
+};
+
+export default class Ghost extends Default {
+  public static readonly spawn_chance: number = 0.1;
+  // @ts-ignore
+  public get name(): 'ghost' {
+    return 'ghost';
+  }
+
+  public get options(): GhostOptions {
+    return {
+      ...super.options,
+      imageUrl: 'balloon.svg',
+      riseDuration: [15000, 20000],
```

Comment:
The rise duration can be a bit longer
```suggestion
      riseDuration: [18000, 21000],
```